boards/arm/nrf54l15dk_nrf54l15/Kconfig.defconfig

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_NRF54L15DK_NRF54L15_CPUAPP
+
+config BOARD
+	default "nrf54l15dk_nrf54l15_cpuapp"
+
+config ENTROPY_GENERATOR
+	default y if CRACEN
+
+endif # BOARD_NRF54L15DK_NRF54L15_CPUAPP
+
+config BT_CTLR
+	default BT
+
+# MPU-based null-pointer dereferencing detection cannot
+# be applied as the (0x0 - 0x400) is unmapped for this target.
+choice NULL_POINTER_EXCEPTION_DETECTION
+	bool
+	default NULL_POINTER_EXCEPTION_DETECTION_NONE
```

Comment:
is this default really needed here ?
afaict, `NULL_POINTER_EXCEPTION_DETECTION_NONE` is already the default choice:
Ref: https://github.com/zephyrproject-rtos/zephyr/blob/979e7a6c5653d3b7e0d0e6c6207b03dfe6f80375/arch/arm/core/cortex_m/Kconfig#L397-L401

Reply:
You're right, it is a default choice. However, my concern is that it may get a different default value if Zephyr config changes. Right now option `NULL_POINTER_EXCEPTION_DETECTION_DWT` is disabled due to some issue. So maybe we could leave it as it is? I can see some other non-Nordic platforms have this entry in their `Kconfig.defconfig` too.

Reply:
> However, my concern is that it may get a different default value if Zephyr config changes.


Every setting in Zephyr can get a new default value.
That's usually not enough reason to change the default of a choice.

What you might do, and which can be accepted, is to specify exception detection in `nrf54l15pdk_nrf54l15_cpuapp_defconfig`, as
```
CONFIG_NULL_POINTER_EXCEPTION_DETECTION_NONE=y
```

Although I really don't see the need in this particular case.

> I can see some other non-Nordic platforms have this entry in their Kconfig.defconfig too.

A lot of other boards does a lot of things in a sub-optimal way.

Reply:
Moved to `_defconfig` file.